### PR TITLE
#205020 Update Aadhaar number in Ip

### PR DESCRIPTION
**Pull request by swapnilphalke03**
### I have ensured that following `Pull Request Checklist` is taken care of before sending this PR
* [x] Code is formatted as per format decided
* [x] Updated acceptance criteria in tracker
* [x] Updated test cases in test-cases-tracker
https://tracker.tekdi.net/issues/205020

### CODE REVIEW

File: lib/common-lib/src/services/facilitatorRegistryService.js [hunk 1]
```diff
@@ -517,4 +517,30 @@ export const exportFacilitatorsCsv = async (params = {}, header = {}) => {
...
+        headers: headers || {}
+      }
+    )
+    if (result?.data) {
+      return result?.data
+    } else {
+      return {}
+    }
+  } catch ({ response, message }) {
+    return {
```
Comment (sagarkoshti1990) [inline lib/common-lib/src/services/facilitatorRegistryService.js:541]: use `handleResponseException`

File: apps/front-end/src/pages/admin/facilitator/View.js [hunk 2]
```diff
@@ -861,6 +911,39 @@ export default function FacilitatorView({ footerLinks }) {
...
+                  value={aadhaarValue}
+                  maxLength={12}
+                  name="numberInput"
+                  onChange={handleAadhaarUpdate}
+                />
+                <AdminTypo.PrimaryButton onPress={updateAadhaar}>
+                  {t("SAVE")}
+                </AdminTypo.PrimaryButton>
+              </HStack>
+              <AdminTypo.H5 mt={3} ml={4} color={"textMaroonColor.400"}>
```
Comment (sagarkoshti1990) [inline apps/front-end/src/pages/admin/facilitator/View.js:940]: show adhaar exist user list

**Review by sagarkoshti1990 — CHANGES_REQUESTED** [review]

File: apps/front-end/src/pages/admin/facilitator/View.js (hunk 2 above)
Comment (swapnilphalke03) [reply apps/front-end/src/pages/admin/facilitator/View.js:981]: Added on line number 982 aadhaarerror

File: apps/front-end/src/pages/admin/facilitator/View.js [hunk 3]
```diff
@@ -19,10 +20,13 @@ import {
   FormControl,
   Input,
   useToast,
+  Pressable,
```
Comment (sagarkoshti1990) [inline apps/front-end/src/pages/admin/facilitator/View.js:23]: remove Pressable

File: apps/front-end/src/pages/admin/facilitator/View.js [hunk 4]
```diff
@@ -170,6 +215,34 @@ export default function FacilitatorView({ footerLinks }) {
...
+      aadhaar_no
+    );
+    if (aadhaarValue.length < 12) {
+      setAadhaarError("AADHAAR_SHOULD_BE_12_DIGIT_VALID_NUMBER");
+    } else if (!data?.success) {
+      setAadhaarError("AADHAAR_NUMBER_ALREADY_EXISTS");
+      setDuplicateUserList(data?.data?.users);
+    } else {
+      setAdhaarModalVisible(false);
+      navigate("/admin/facilitator");
```
Comment (sagarkoshti1990) [inline apps/front-end/src/pages/admin/facilitator/View.js:242]: remove `navigate` and update `data` state and `adhar_no`

File: apps/front-end/src/pages/admin/facilitator/View.js [hunk 5]
```diff
@@ -861,6 +953,48 @@ export default function FacilitatorView({ footerLinks }) {
             ))}
           </VStack>
         </VStack> */}
+        <Modal
+          isOpen={adhaarModalVisible}
+          onClose={() => setAdhaarModalVisible(false)}
```
Comment (sagarkoshti1990) [inline apps/front-end/src/pages/admin/facilitator/View.js:958]: Remove `onClose`

**Review by sagarkoshti1990 — CHANGES_REQUESTED** [review]

**Review by sagarkoshti1990 — APPROVED** [review]

**Pull request merged**